woocommerce/woocommerce
# remove the generic incompatible notice from Checkout block.

Removes the generic incompatibility notice from Checkout buts keeps the payment and plugins ones.

Closes https://github.com/woocommerce/woocommerce/issues/47192



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Select Checkout block in the post editor, make sure the blue incompatibility notice is not visible.
2. Enable a plugin that isn't compatible like One Page Checkout, Checkout Field Editor.
3. Make sure the yellow notice is visible.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
Removes the incompatible notice from Checkout.

</details>
